docs/integrations/sources/google-ads.md

```diff
@@ -83,6 +85,12 @@ If for any reason the request gets denied, let us know and we will be able to un
 
 The Google Ads Query Language can query the Google Ads API. Check out [Google Ads Query Language](https://developers.google.com/google-ads/api/docs/query/overview)
 
+## Understanding Click View Stream Date Range 
+
+Queries including ClickView must have a filter limiting the results to one day and can be requested for dates back to 90 days before the time of the request.
```

Comment:
This doc is not helpful to the user - do they need to take action based on this? Or is this just letting them know? 

If the latter, we should frame it as an FYI e.g: " Due to constraints from the Google Ads API, the `Clickview` stream retrieves data one day at a time and can only retrieve data newer than 90 days ago"

What action does the user need to take based on this doc?